components/org.wso2.carbon.identity.api.server.certificate.validation.management/org.wso2.carbon.identity.api.server.certificate.validation.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/certificate/validation/management/v1/factories/CertificateValidationApiServiceFactory.java

```diff
@@ -35,5 +35,5 @@ public class CertificateValidationApiServiceFactory {
      */
     public static CertificateValidationApiService getCertificateValidationApi() {
         return SERVICE;
-    }
+   }
```

Comment:
Let's remove this modification

Reply:
fixed with https://github.com/wso2/identity-api-server/pull/835/commits/913b13b421bc59ef42c4a6446c5e11912b46badf